### DISCUSSION TITLE
What is an alternative to the experimenal revalidate feature?

### DISCUSSION THREAD

**Discussion by bryandandan** [vercel/next.js#17191] [Help] [answered]
Since the revalidate feature is still experimental what would be an alternative? It is said in the documentation to use getStaticProps for headless CMS. However, when the data in the cms changes, you have to re-deploy the app to get the new changes every time.

**Comment by jamesmosier** [comment, marked as answer]
Hi @bryandandan. `revalidate` is not experiment anymore and is in the official [docs](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation)! So definitely give it a try and it should work well for your use case 😄

**Reply by bryandandan** [reply]
Correct. I was using a previous version of next js. I fixed it by updating to the latest. Thanks!
